CONTRIBUTING.md

```diff
@@ -142,14 +142,14 @@ For imports you should use the following format (_without_ the comments)
 ```go
 import (
   // stdlib
-  "encoding/json"
   "fmt"
 
   // local packages
   "code.gitea.io/gitea/models"
   "code.gitea.io/sdk/gitea"
 
   // external packages
+  "github.com/goccy/go-json"
```

Comment:
This should be moved above, and referenced as `"code.gitea.io/gitea/modules/json"`